src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -880,13 +889,13 @@ def aks_update(cmd,     # pylint: disable=too-many-statements,too-many-branches,
                enable_secret_rotation=False,
                disable_secret_rotation=False,
                rotation_poll_interval=None,
-               enable_disk_driver=None,
+               enable_disk_driver=False,
```

Comment:
I think the problem is maybe on update, we should set `None` if `enable_disk_driver` or `disable_disk_driver` is not specified? will this change the original value if `enable_disk_driver` or `disable_disk_driver` is not specified?